src/components/breadcrumbs/_macro.spec.js

```diff
@@ -5,125 +5,130 @@ import * as cheerio from 'cheerio';
+            });
+        });
     });
-
-    it('has a "chevron" icon for each breadcrumb item', () => {
-        const faker = templateFaker();
-        const iconsSpy = faker.spy('icon');
-
-        faker.renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_MINIMAL);
-
-        const iconTypes = iconsSpy.occurrences.map((occurrence) => occurrence.iconType);
-        expect(iconTypes).toEqual(['chevron', 'chevron']);
+    describe('GIVEN: Params: itemsList', () => {
+        describe('WHEN: itemClasses param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided style classes', () => {
+                expect($('.ons-breadcrumbs__item:first').hasClass('item-extra-class')).toBe(true);
+                expect($('.ons-breadcrumbs__item:first').hasClass('item-another-extra-class')).toBe(true);
+            });
+        });
+        describe('WHEN: linkClasses param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders link with provided style classes', () => {
+                expect($('.ons-breadcrumbs__link').hasClass('link-extra-class')).toBe(true);
+                expect($('.ons-breadcrumbs__link').hasClass('link-another-extra-class')).toBe(true);
+            });
+        });
+        describe('WHEN: id param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided id', () => {
+                expect($('.ons-breadcrumbs__link').attr('id')).toBe('first-breadcrumb');
+            });
+        });
+        describe('WHEN: url param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided url link', () => {
+                const url = mapAll($('.ons-breadcrumbs__link'), (node) => node.attr('href'));
+                expect(url).toEqual(['https://example.com/']);
+            });
+        });
+        describe('WHEN: text param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item link with provided text', () => {
+                const label = mapAll($('.ons-breadcrumbs__link'), (node) => node.text().trim());
+                expect(label).toEqual(['Home']);
+            });
+        });
+        describe('WHEN: attributes param is provided', () => {
+            const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS_ITEM_PARAMS));
+            test('THEN: renders item with provided attributes', () => {
+                const testValuesA = mapAll($('.ons-breadcrumbs__link'), (node) => node.attr('data-a'));
+                expect(testValuesA).toEqual(['123']);
```

Comment:
This seems to be only testing the attributes on the first breadcrumb where the original test tested the ones on the second one too